distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -58,6 +59,28 @@ Text.length =
         @Tail_Call count (accum + 1) iterator.next
     count 0 iterator.next
 
+## Returns a new `Text` object with the characters in the reverse order of the input.
+
+   ! What is a Character?
+     A character is defined as an Extended Grapheme Cluster, see Unicode
+     Standard Annex 29. This is the smallest unit that still has semantic
+     meaning in most text-processing applications.
+
+   > Example
+     Reverse the text "Hello, world!".
+
+       "Hello, world!".reverse
```

Comment:
```suggestion
   > Example
     Reverse the text "Hello, world!".

         "Hello, world!".reverse
```

I think the code in examples should also indented 4 spaces relative to the caption.

Reply:
My bad, missed that. Should be fixed now!